devito/ir/iet/nodes.py

```diff
@@ -353,6 +356,19 @@ def __init__(self, expr):
         super(Increment, self).__init__(expr, '+')
 
 
+class Break(Expression):
+    """A node representing a break statement"""
+
+    is_Break = True
+
+
+class Reduce(Expression):
```

Comment:
Difference with `class Increment` ?

Reply:
The "Reduce" class will be removed from the pull request.